zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -392,7 +413,7 @@ fn step_convert(
         gas_left: step.gas_left.0,
         gas_cost: step.gas_cost.as_u64(),
         opcode: Some(step.op),
-        memory_size: step.memory_size as u64 / 32, /* memory size in word */
+        memory_size: step.memory_size as u64,
```

Comment:
I think we can have `memory_word_size` directly here, then save the function above.

Reply:
I would prefer to have `memory_size` (in bytes) in the witness so that in the `MSIZE` gadget we can directly assign `step.memory_size` to the msize `value` cell (see [here](https://github.com/appliedzkp/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/execution/msize.rs#L79)) without conversion (e.g., multiply by 32). Let me know if this makes sense.